components/ui/loader/Loader.tsx

```diff
@@ -8,8 +8,7 @@ interface LoaderProps {
 
 const Loader: React.FC<LoaderProps> = ({ isLoading = false, hasViewHeader = false }) => {
   const osName = Platform.OS;
-  const screenHeight = Dimensions.get('window').height - (hasViewHeader ? 192 : 0); // height of header * 2;
-
+  const screenHeight = Dimensions.get('window').height - (hasViewHeader ? 96 : 0); // height of header * 2;
```

Comment:
why we have chandeg that to 96, two times less that 192 ?

Reply:
will change comment. This is the error i talk about that laoder was to small to cover button at the bottom